### PR TITLE
self.gui.show_animated_image

### PR DESCRIPTION
**Pull request by JarbasAl**
Adds some missing stuff from SkillGui docstrings

adds a new method and qml template to show animated images (gif files)

### CODE REVIEW

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Failed ([Results](https://reports.mycroft.ai/core/PR-2616))

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Failed ([Results](https://reports.mycroft.ai/core/PR-2616))

**Comment by forslund** [comment]
Good addition and docstrings are spot on. Also nice catch with the `{}` > `None`

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/core/PR-2616))

**Comment by AIIX** [comment]
+1 awesome works fine for me

**Comment by krisgesling** [comment]
Excellent, thanks Jarbas, and everyone else for reviewing :)

File: mycroft/enclosure/gui.py [hunk 1]
```diff
@@ -111,14 +111,15 @@ def clear(self):
         self.skill.bus.emit(Message("gui.clear.namespace",
                                     {"__from": self.skill.skill_id}))
 
-    def send_event(self, event_name, params={}):
+    def send_event(self, event_name, params=None):
```
Comment (AIIX) [inline mycroft/enclosure/gui.py:114]: This causes mycroft-gui to crash, the session model does not like receiving None type params and then receiving objects, please revert this.

**Comment by AIIX** [comment]
@krisgesling as per review this should have remained open, but since it was already merged, please find the revert / fix in the following PR https://github.com/MycroftAI/mycroft-core/pull/2617

File: mycroft/enclosure/gui.py (hunk 1 above)
Comment (forslund) [reply mycroft/enclosure/gui.py:114]: The none is changed to `{}` a couple of lines down so it shouldn't be an issue...

**Pull request merged**